packages/plugins/rubygem-newt/rubygem-newt.spec

```diff
@@ -1,101 +1,75 @@
-# template: scl
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-
+# template: default
 %global gem_name newt
 %global gem_require_name %{gem_name}
 
-Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.9.7
-Release: 3%{?dist}
+Name: rubygem-%{gem_name}
+Version: 1.0.1
+Release: 1%{?dist}
 Summary: Ruby bindings for newt
-Group: Development/Languages
 License: MIT
 URL: https://github.com/theforeman/ruby-newt
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
+BuildRequires: newt-devel
```

Comment:
I'd prefer to keep it below (like it was before your PR), but that's more a personal preference than a requirement ;)